arch/arm64/include/irq.h

```diff
@@ -161,6 +161,10 @@
 #define ARM64_CONTEXT_REGS  (38)
 #define ARM64_CONTEXT_SIZE  (8 * ARM64_CONTEXT_REGS)
 
+#if ARM64_CONTEXT_SIZE % 16 != 0
```

Comment:
```suggestion
#if XCPTCONTEXT_SIZE % 16 != 0
```
and move after line 233

Reply:
Done.